packages/yoastseo/GLOSSARY.md

```diff
@@ -0,0 +1,184 @@
+# YoastSEO.js Glossary
+
+A comprehensive glossary of terms and concepts used in YoastSEO.js.
+
+## Core Concepts
+
+### <a name="paper"></a>Paper
+A value object that encapsulates all content and metadata to be analyzed. The Paper object is immutable and serves as the primary input for all analyses.
+
+**Properties include:**
+- text: The main content to analyze
+- title: The SEO title or page title
+- keyword: The focus keyphrase
+- description: Meta description
+- url: The slug/URL of the page
+- locale: Language code (e.g., en_US)
+- permalink: Full URL
+
+**Example:**
+```javascript
+const paper = new Paper("This is the main content", {
+    title: "Example Title",
+    keyword: "example",
+    description: "This is a meta description",
+    url: "example-page",
+    locale: "en_US"
+});
+```
+
+### <a name="assessment"></a>Assessment
+A single analysis unit that evaluates one specific aspect of content. Each assessment:
+- Has a specific purpose (e.g., keyword density analysis)
+- Produces a score (0-100)
```

Comment:
```suggestion
- Produces a score (0-9)
```

Reply:
Is that meant to be `0-90`?

Reply:
No `0-9` is actually correct 😄

Reply:
(well, in extreme cases, you can have negative scores as well, e.g. if you do not enter a focus keyphrase)